### ISSUE TITLE
All tags in each release instead only commits between tags

### ISSUE THREAD

**Issue by papanito** [github-tools/github-release-notes#279]
I want to generate the changelog for all releases I have running this

```bash

**Comment by robert-lacagnina** [comment]
Yeah I'll add to the I've seen similar behavior on `0.17.3`. I tried to generate release notes between two specific tags which only have about 3 commits and I got all my commits listed in the changelog. Downgrading to `0.17.2` resolved the issue and I saw only commits between tags as expected.

**Comment by snahelou** [comment]
Same problem for me. I rollback to `0.17.1`.

**Comment by cjbarth** [comment]
@airshipbot, @v1v  I believe the changes I've made in my fork correct for this problem. Please let me know if they don't. https://github.com/cjbarth/github-release-notes/releases/tag/0.18.0
